src/edge/edgeManager.ts

```diff
@@ -609,6 +609,22 @@ export class EdgeManager {
         return undefined;
     }
 
+    private async validateSolutionName(name: string, parentPath?: string): Promise<string | undefined> {
+        if (!name) {

```

Comment:
name could be spaces? So we need to trim the spaces

Reply:
Oh, yes, I will trim it.